styles/_variables.scss

```diff
@@ -8,3 +8,4 @@ $half-whitespace-unit: $standard-whitespace-unit / 2;
 
 /* layout */
 $standard-gutter-width: 10px;
+$standard-footer-breakpoint: 599px;
```

Comment:
Move responsive part to workwise until we bring the responsive breakpoints to styleguide. Which we should do..but not right now.